tests/test_scrape.py

```diff
@@ -71,5 +72,5 @@ def test_data():
         json=sample_data
     )
 
-    prom_data = data(enrollment, token, month='2017-03')
-    assert prom_data.count('azure_costs') == 4
+    prom_data = data(enrollment, token, metric_name, month='2017-03')
+    assert prom_data.count(metric_name) == 4
```

Comment:
I have no idea what the `4` is supposed to mean or how I can derive this number from the input data. Reading through the data frame code was not too enlightening, but then again I am no expert.

I would have expected a check that the metric returns the actual money that was spent, which I would deduce from the `ExtendedCost` fields in the sample response...